playground/config/transformer-now.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
```

Comment:
Do we want to offer default transformers or do we require people to always write their own?

Reply:
I thought about it and I think for now it does not make sense to have a default transformers. It's also a bit tricky to provide a good default.
For now, I would just provide examples as documentation, so people can just copy-paste it and update it to their needs.